_DeploymentAndDistroPackaging/ansible/doc/development.md

```diff
@@ -1,8 +1,16 @@
 # Development Mode
-This playbooks installs ciao from the latest ciao packaged for your distro.
+By default, these playbooks install ciao on ciao on production environments from
```

Comment:
here we have a repition of ciao on, e.g., 'ciao on ciao on'.  Is this intentional?

'install ciao in production environments' would be better I think